app/services/search/user.rb

```diff
@@ -12,31 +13,24 @@ def search_documents(params:)
         query_hash = Search::QueryBuilders::User.new(params).as_hash
 
         results = search(body: query_hash)
-        hits = results.dig("hits", "hits").map do |user_doc|
+        hits = results.dig("hits", "hits")
+        paginated_results = paginate_hits(hits, params)
+
+        hits = paginated_results.map do |user_doc|
           prepare_doc(user_doc.dig("_source"))
         end
-        paginate_hits(hits, params)
+
+        {
+          users: hits
+        }.merge(metadata(results, params)).with_indifferent_access
       end
 
       private
 
       def prepare_doc(hit)
-        {
-          "user" => {
-            "username" => hit["username"],
-            "name" => hit["username"],
-            "profile_image_90" => hit["profile_image_90"]
-          },
-          "title" => hit["name"],
-          "path" => hit["path"],
-          "id" => hit["id"],
-          "class_name" => "User",
-          "positive_reactions_count" => hit["positive_reactions_count"],
-          "comments_count" => hit["comments_count"],
-          "badge_achievements_count" => hit["badge_achievements_count"],
-          "last_comment_at" => hit["last_comment_at"],
-          "roles" => hit["roles"]
-        }
+        SERIALIZER_CLASS.attributes_to_serialize.keys.reduce({}) do |m, k|
```

Comment:
I'm sure this change breaks some place. Still need to find out.

Reply:
I don't think we can do this bc we have key combos like `"title" => hit["name"],` and `"name" => hit["username"]`  that don't match up exactly but are needed for the main feed.